server/handlers/handlers.go

```diff
@@ -352,7 +352,11 @@ func (ah *ActivityHandler) handleUpdatedActivity(activityIds msteams.ActivityIds
 					ah.plugin.GetAPI().LogError("Unable to recover the post", "postID", postInfo.MattermostID, "error", err)
 					return discardedReasonOther
 				}
-				post, _ = ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
+				post, err = ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
+				if err != nil {
```

Comment:
I think we need to rename as `appErr` here since the plugin API doesn't return the `error` interface but an instance of `model.AppError` making this always true. (Love the inline linter comments!)

Reply:
updated and fixed ci